helm/cluster-apps-operator/values.yaml

```diff
@@ -51,6 +54,8 @@ release:
         chart-operator:
           chart:     "chart-operator"
           namespace: "giantswarm"
+        cloud-provider-openstack-app:
+          hasClusterValuesSecret: true
```

Comment:
This override specifies which apps require the cluster values secret so that we don't pass it into every app.